### PR TITLE
Create 0016-soto.md

### PR DESCRIPTION
**Pull request by adam-fowler**
@tomerd as requested

### CODE REVIEW

File: proposals/0016-soto.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# Soto for AWS
+
+* Proposal: [SSWG-NNNN](NNNN-filename.md)
+* Authors: [Adam Fowler](https://github.com/adam-fowler), [Fabian Fett](https://github.com/fabianfett), [Jonathan McAllister](https://github.com/jonnymacs), [Joe Smith](https://github.com/Yasumoto), [Yuki Takei](https://github.com/noppoman)
+* Review Manager: TBD
```
Comment (kmahar) [inline proposals/0016-soto.md]:
```suggestion
* Review Manager: [Kaitlin Mahar](https://github.com/kmahar)
```

File: proposals/0016-soto.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
+# Soto for AWS
+
+* Proposal: [SSWG-NNNN](NNNN-filename.md)
```
Comment (kmahar) [inline proposals/0016-soto.md]:
```suggestion
* Proposal: [SSWG-0016](0016-soto.md)
```

File: proposals/0016-soto.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+* Authors: [Adam Fowler](https://github.com/adam-fowler), [Fabian Fett](https://github.com/fabianfett), [Jonathan McAllister](https://github.com/jonnymacs), [Joe Smith](https://github.com/Yasumoto), [Yuki Takei](https://github.com/noppoman)
+* Review Manager: TBD
+* Status: **Implemented**
```
Comment (kmahar) [inline proposals/0016-soto.md]:
```suggestion
* Status: Active Review 28 Oct 2020 ..< 12 Nov 2020
```

**Review by kmahar — COMMENTED** [review]
Hi @adam-fowler, I'll be the review manager for this proposal. 

I've made suggestions with updates to the proposal to reflect the number, review manager, and review period dates. Let me know if these look good to you, and if so I'll go ahead and merge this and then make a post on the forums to solicit community feedback.

Thanks!

**Comment by adam-fowler** [comment]
Hi @kmahar sounds good

**Review by kmahar — APPROVED** [review]

**Pull request merged**